src/orca-jedi/state/StateParameters.h

```diff
@@ -27,6 +27,7 @@ class OrcaStateParameters : public oops::Parameters {
   oops::OptionalParameter<bool> analyticInit{"analytic initialisation", this};
   oops::OptionalParameter<std::string> outputNemoFieldFile{
     "output nemo field file", "", this};
+  oops::OptionalParameter<bool> setGmask{"set gmask", this};
```

Comment:
I keep forgetting what a gmask is sorry! Is it short for grid mask? I know it is distinct from the land sea mask. Could we maybe stick a description into this parameter constructor. e.g as used for some GeometryParameters. (the "description" constructor argument here https://github.com/JCSDA-internal/oops/blob/463e67a58b0f7a966bed438ea08533db9d795b3d/src/oops/util/parameters/RequiredParameter.h#L77)

Otherwise everything seems good to me.

Reply:
gmask = geometry mask. It is currently only used by BUMP in SABER which uses it to generate a coarse finite element grid. It's possible that it may get used by other elements of SABER in future for example diffusion based covariance code.

I added a description section including a bit more info and included an indication when set_gmask needs to true.